keyboards/plaid/keymaps/jerksen/keymap.c

```diff
@@ -0,0 +1,412 @@
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Exit |      |      |   A  |   O  |             |   E  |   U  |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+
+[_PLOVER] = LAYOUT_plaid_grid(
+    KC_1,    KC_1,    KC_1,    KC_1,    KC_1,    KC_1,    KC_1,    KC_1,    KC_1,    KC_1,    KC_1,    KC_1   ,
+    XXXXXXX, KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC,
+    XXXXXXX, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,
+    EXT_PLV, XXXXXXX, XXXXXXX, KC_C,    KC_V,    XXXXXXX, XXXXXXX, KC_N,    KC_M,    XXXXXXX, XXXXXXX, XXXXXXX
+),
+
+/* Adjust (Lower + Raise)
+ * ,-----------------------------------------------------------------------------------.
+ * |Reset |      |      |      |      |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      | Debug| MuMod|Aud on|Audoff|AGnorm|AGswap|Qwerty|Colemk|Dvorak|Plover|      |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |Voice-|Voice+|Mus on|Musoff|MIDIon|MIDIof|      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_ADJUST] = LAYOUT_plaid_grid(
+    RESET,   LED_1,   LED_2,   LED_3,   LED_4,   LED_5,   LED_6,   LED_7,   LED_8,    LED_9,   LED_0,   _______,
+    _______, DEBUG,   MU_MOD,  AU_ON,   AU_OFF,  AG_NORM, AG_SWAP, _______, _______,  _______, _______, _______,
+    _______, MUV_DE,  MUV_IN,  MU_ON,   MU_OFF,  MI_ON,   MI_OFF,  TERM_ON, TERM_OFF, _______, _______, _______,
+    _______, QWERTY,  COLEMAK, DVORAK, _______,  _______, _______, _______, PLOVER,   _______, _______, _______
+)
+
+
+};
+
+//Setup config struct for LED
+typedef union {
+  uint32_t raw;
+  struct {
+    uint8_t  red_mode :8;
+    uint8_t  green_mode :8;
+  };
+} led_config_t;
+led_config_t led_config;
+
+//Set leds to saved state during powerup
+void keyboard_post_init_user(void) {
+  // Call the post init code.
+  led_config.raw = eeconfig_read_user();
+
+  if(led_config.red_mode == LEDMODE_ON) {
+      writePinHigh(LED_RED);
+  }
+
+  if(led_config.green_mode == LEDMODE_ON) {
+      writePinHigh(LED_GREEN);
+  }
+}
+
+void eeconfig_init_user(void) {  // EEPROM is getting reset! 
+  led_config.raw = 0;
+  led_config.red_mode = LEDMODE_ON;
+  led_config.green_mode = LEDMODE_MODS;
+      eeconfig_update_user(led_config.raw);
+  eeconfig_update_user(led_config.raw);
+}
+
+uint32_t layer_state_set_user(uint32_t state) {
```

Comment:
this matches the default config and I'd like to leave it as is.

Reply:
Maybe, but it's wrong.  It's no longer a uint32_t, but is typedef'ed, so that you can actually limit the number of layers that you can use.

And the default should be fixed/updated.